### DISCUSSION TITLE
Annotations clipping through one another.

### DISCUSSION THREAD

**Discussion by ev88** [google/model-viewer#2947] [Q&A]
I'm running into an issue with the annotation hotspots (circles) overlapping the text box of the active annotation at certain angles.  I have toyed around with z-index on these items, but it hasn't had any effect.

https://tribesmodels.com/annotation/smartphone/

![Screenshot 2021-11-01 141731](https://user-images.githubusercontent.com/6477567/139721838-b650ea7e-9d9d-4d2c-a4d4-7a953385da99.png)

**Comment by elalish** [comment]
That's very interesting, I can repro. We don't do anything special there, so I'd guess this is just normal CSS weirdness. Have you tried changing the order of the hotspot elements? You may want to take a look at what causes browsers to generate new layers, as  that seems like where the problem is.
